output-diary.c:55:1: warning: control reaches end of non-void function [-Wreturn-type]
}
^
output-diary.c:72:1: warning: control reaches end of non-void function [-Wreturn-type]
}
^
output-text.c:65:1: warning: control reaches end of non-void function [-Wreturn-type]
}
^
output-text.c:89:1: warning: control reaches end of non-void function [-Wreturn-type]
}
^
--- src/output-diary.c.orig	2001-05-05 07:40:44.000000000 -0500
+++ src/output-diary.c	2018-11-05 07:36:13.000000000 -0600
@@ -36,7 +36,7 @@
 extern int dump_file(FILE * from, FILE * to);
 
 /* out_diary_fputnote - outputs a note in diary format to a file */ 
-int out_diary_fputnote(dnote d, FILE * to)
+void out_diary_fputnote(dnote d, FILE * to)
 {
   int i;
 
@@ -56,7 +56,7 @@
 
 /* out_diary_special_fputnote - outputs a note in diary format to a file
    but with the note text in a file */
-int out_diary_special_fputnote(dnote d, FILE * to, FILE * source)
+void out_diary_special_fputnote(dnote d, FILE * to, FILE * source)
 {  
 
   fprintf(to, "<note>\n<date>%s</date>\n",dencode_date(d));
--- src/output-text.c.orig	2018-11-05 07:33:38.000000000 -0600
+++ src/output-text.c	2018-11-05 07:36:03.000000000 -0600
@@ -35,7 +35,7 @@
 
 
 /* out_text_fputnote - outputs a note in plain text format to a file */ 
-int out_text_fputnote(dnote d, FILE * to)
+void out_text_fputnote(dnote d, FILE * to)
 {
   int i, j;
   char * date;
@@ -66,7 +66,7 @@
 
 
 /* out_text_fputsummary - outputs note summary in plain text format to file */ 
-int out_text_fputsummary(dnote d, FILE * to, int id)
+void out_text_fputsummary(dnote d, FILE * to, int id)
 {
   int j;
   char * date;
--- src/output.h.orig	2018-11-05 07:33:38.000000000 -0600
+++ src/output.h	2018-11-05 07:36:39.000000000 -0600
@@ -25,10 +25,10 @@
 #ifndef __OUTPUT_H__SAND__
 #define __OUTPUT_H__SAND__
 
-int out_diary_fputnote(dnote d, FILE * to);
-int out_diary_special_fputnote(dnote d, FILE * to, FILE * source);
+void out_diary_fputnote(dnote d, FILE * to);
+void out_diary_special_fputnote(dnote d, FILE * to, FILE * source);
 
-int out_text_fputnote(dnote d, FILE * to);
-int out_text_fputsummary(dnote d, FILE * to, int id);
+void out_text_fputnote(dnote d, FILE * to);
+void out_text_fputsummary(dnote d, FILE * to, int id);
 
 #endif
